Microsoft.Dynamics365.UIAutomation.Api.UCI/WebClient.cs

```diff
@@ -5300,7 +5300,7 @@ internal void ThinkTime(TimeSpan timespan)
             ThinkTime((int)timespan.TotalMilliseconds);
         }
 
-        internal void Dispose()
+        public void Dispose()

```

Comment:
Why internal? Browser stills open after tests is finish. (on Resharper Tests Runner)